docs/TOC.md

```diff
@@ -54,6 +54,7 @@
 * [Pinning (frozen) of Columns/Rows](grid-functionalities/frozen-columns-rows.md)
 * [Row Selection](grid-functionalities/Row-Selection.md)
 * [Tree Data Grid](grid-functionalities/Tree-Data-Grid.md)
+* [Row Based Editing Plugin](grid-functionalities/Row-based-edit.md)
```

Comment:
doesn't need to be changed but just as a side note, I want to use only lowercase for new docs. I haven't changed them all because I'm on Windows and Git doesn't detect case change by default (there's some extra git command to run which I forgot), so just a reminder for future docs...